Engineering.md

```diff
@@ -79,47 +93,58 @@
 - I am actively introspective and take time to work on my personal and professional development.
 - I happily take responsibility for my own mistakes and I learn from the mistakes of others.
 
+## Team Lead
+
+> I am trusted to autonomously lead people through complex problems with a proven record of delivering ideas from inception to their successful completion.
+> A Team Lead emcompasses all of the qualities listed in [Leadership](Leadership.md) and is usually combined with either a Senior Software Engineer or Solution Architect role.
+
+### I am trusted to lead a team of diverse technical roles (from Architects, Engineers, Designers, and Testers) to ship shaped pitches to production
+
+- I am a well-rounded leader of people, known for having a collaborative leadership style.
+- I am proficient at leading software projects using agile practices.
+- I am comfortable with transparently assessing risk, making recommendations, escalating appropriately, and dealing with the consequences along the way.
+- I can confidently and charismatically pitch ideas, positively influencing my team to take decisive action based on reaching consensus.
+- I am an adept communicator, and can effectively steer technical and non-technical conversations to positive outcomes over any medium.
+
+## Engineering Manager
```

Comment:
**First impression**: These characteristics form a great description of my ideal manager; love it 💙💙💙

**Second thought**: Does this description aim to answer these kinds of more specific questions?
- Do Managers at Octopus influence architecture? Write code? Are they ultimately responsible for any cross-cutting concerns like security? (etc)
- Do Managers at Octopus influence product? (see: betting table)
- Do Managers hire/fire/do salary reviews, etc?

Reply:
> Do Managers at Octopus influence architecture? Write code? Are they ultimately responsible for any cross-cutting concerns like security? (etc)

In our current situation, yes, but I think that is a byproduct of us historically being Solution Architects. In this section, I want to focus on the unique value proposition of an Engineering Manager, explaining their first priorities, and if they contribute extra value outside this, that could be OK.

> Do Managers at Octopus influence product? (see: betting table)

I have the same sentiment here also. Our current engineering managers are involved in the betting table more as a product of history and capability at the moment, and perhaps that will change over time? One unique value we contributed to the betting table was capacity planning: which bets can we actually achieve with the people we have in each tribe?

> Do Managers hire/fire/do salary reviews, etc?

Yes, I will definitely add something around this. This is part of where I was going with identifying and dealing with dysfunction.